dali/python/tf_plugin/setup.py.in

```diff
@@ -77,7 +77,7 @@ class CustomInstall(install, object):
 
         super(CustomInstall, self).run()
 
-setup(name='nvidia-dali-tf-plugin',
+setup(name='nvidia-dali-tf-plugin@DALI_FLAVOR@',
```

Comment:
need to update dependency with nvidia-dali to have @DALI_FLAVOR@ as well

Reply:
Done